templates/default.yaml

```diff
@@ -246,6 +246,27 @@ containerd:
 #   owner: "root:root"
 #   permissions: 644
 #   overwrite: false
+# # `yq` creates or edits a file in the guest filesystem by using `${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent yq`.
+# # `${LIMA_CIDATA_GUEST_INSTALL_PREFIX}/bin/lima-guestagent yq` has the same functionality as https://github.com/mikefarah/yq.
+# # The file specified by `path` will be updated with the specified `expression` using `yq --inplace`.
+# # If the file does not exist, it will be created using `yq --null-input`.
+# # `format` is optional and defaults to "auto", which is passed to `yq` by `--input-format <format>`, `--output-format <format>`.
+# # One of the following is expected to work:
+# #   "auto", "a", "ini", "i", "json", "j", "props", "p", "xml", "x", "yaml", "y"
+# # `yq` v4.47.1 can write .ini, .json, .properties, .xml, .yaml, and .yml files.
```

Comment:
User does not need to, and not even able to, care about the version of the yq library